frame/babe/src/lib.rs

```diff
@@ -521,6 +520,12 @@ impl<T: Config> pallet_session::ShouldEndSession<T::BlockNumber> for Pallet<T> {
 }
 
 impl<T: Config> Pallet<T> {
+	/// The configuration for the current epoch.
+	/// Should never be `None` as it is initialized in genesis.
+	pub fn epoch_config() -> Option<BabeEpochConfiguration> {
+		EpochConfig::<T>::get()
+	}
+
```

Comment:
replaced the getter with another getter?

Reply:
Because the idea of this PR is that the automatically generated getter is deprecated.